VirtualReality/Widgets/qMRMLVirtualRealityView.cxx

```diff
@@ -456,9 +462,65 @@ void qMRMLVirtualRealityViewPrivate::doOpenVirtualReality()
       this->Camera->GetViewPlaneNormal(this->LastViewDirection);
       this->Camera->GetViewUp(this->LastViewUp);
       this->Camera->GetPosition(this->LastViewPosition);
+
+      int disabledModify = this->MRMLVirtualRealityViewNode->StartModify();
```

Comment:
I don't see modification of this->MRMLVirtualRealityViewNode in updateTransformNodeWithControllerPose calls. I think we should call Start/EndModify for the transform nodes instead.

Reply:
Right, that makes a lot more sense. Will do.